tests/unit/dataactvalidator/test_c11_cross_file.py

```diff
@@ -15,39 +15,58 @@ def test_column_headers(database):
 def test_success(database):
     """ Unique PIID, ParentAwardId from file C exists in file D1 during the same reporting period. """
 
-    af = AwardFinancialFactory(piid='some_piid', parent_award_id='some_parent_award_id')
+    af = AwardFinancialFactory(piid='some_piid', parent_award_id='some_parent_award_id',
+                               transaction_obligated_amou='12345')
     ap = AwardProcurementFactory(piid='some_piid', parent_award_id='some_parent_award_id')
 
     assert number_of_errors(_FILE, database, models=[af, ap]) == 0
 
-    af = AwardFinancialFactory(piid='some_piid', parent_award_id='some_parent_award_id')
+    af = AwardFinancialFactory(piid='some_piid', parent_award_id='some_parent_award_id',
+                               transaction_obligated_amou='12345')
     ap_1 = AwardProcurementFactory(piid='some_piid', parent_award_id='some_parent_award_id')
     ap_2 = AwardProcurementFactory(piid='some_piid', parent_award_id='some_parent_award_id')
 
     assert number_of_errors(_FILE, database, models=[af, ap_1, ap_2]) == 0
 
-    af = AwardFinancialFactory(piid=None, parent_award_id='some_parent_award_id')
+    af = AwardFinancialFactory(piid=None, parent_award_id='some_parent_award_id',
+                               transaction_obligated_amou='12345')
     ap = AwardProcurementFactory(piid='some_piid', parent_award_id='some_parent_award_id')
 
     assert number_of_errors(_FILE, database, models=[af, ap]) == 0
 
-    af = AwardFinancialFactory(piid='some_piid', parent_award_id=None)
+    af = AwardFinancialFactory(piid='some_piid', parent_award_id=None, transaction_obligated_amou='12345')
     ap = AwardProcurementFactory(piid='some_piid', parent_award_id=None)
 
     assert number_of_errors(_FILE, database, models=[af, ap]) == 0
 
+    # Not perform when no transaction obligated amount value in the field
+    af = AwardFinancialFactory(piid='some_piid', parent_award_id='some_parent_award_id',
+                               allocation_transfer_agency=None, transaction_obligated_amou=None)
```

Comment:
It tests if the `transction_obligated_award_id is NULL`, it should pass even with af.PIID is not the same with ap.PIID. We don't need that, but I added to show that it meet the acceptance criteria only performed on rows that have a `transction_obligated_award_id`.

I can remove that if you think we don't need to test that.